contracts/prize-pool/PrizePool.sol

```diff
@@ -719,11 +719,10 @@ abstract contract PrizePool is PrizePoolInterface, OwnableUpgradeable, Reentranc
     )
   {
     (uint256 exitFee, uint256 _burnedCredit) = _calculateEarlyExitFeeLessBurnedCredit(from, controlledToken, amount);
```

Comment:
You must assign to burnedCredit, not declare _burnedCredit.  The return value never gets assigned.